libs/application/core/src/lib/dataProviderUtils.ts

```diff
@@ -9,10 +9,10 @@ import {
 } from '../types/BasicDataProvider'
 import { FormatMessage, StaticText } from '../types/Form'
 import { coreErrorMessages } from './messages'
-import { isString } from 'lodash'
+import { isString, isObject } from 'lodash'
```

Comment:
There should be an eslint rule that forbids this. But we don't want to import destructed objects from lodash since it'll import the whole lodash library and increase the bundle size.

This is preferred due to a much lower bundle size:
```typescript
import isString from 'lodash/isString'
```